uint32 session_id
uint32 camera_id
uint32 chand_id
time ts
Float32XYZ pos
float32 confidence
string[] gestures